src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,165 @@
-""" This is a dummy example """
+import json
+import logging
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import requests
+import time
+
+
+logger = logging.getLogger(__name__)
+
+logger.level = logging.INFO
+# flake8: noqa E501
+# import time
+# import pandas
+
+API_URL: str = "https://climate-api.open-meteo.com/v1/climate?"
+
+COORDINATES: dict = {
+    "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+    "London": {"latitude": 51.507351, "longitude": -0.127758},
+    "Rio": {"latitude": -22.906847, "longitude": -43.172896},
+}
+VARIABLES: str = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
+MODELS: str = (
+    "CMCC_CM2_VHR4,FGOALS_f3_H,HiRAM_SIT_HR," "MRI_AGCM3_2_S,EC_Earth3P_HR,MPI_ESM1_2_XR,NICAM16_8S"
+)
+
+
+def get_data_meteo_api(city: str, start_date: str, end_date: str):
```

Comment:
You can add output type with `get_data_meteo_api(city: str, start_date: str, end_date: str) -> dict:`